### DISCUSSION TITLE
[FEATURE]: findAllListingsActive should optionally be able to specify includes

### DISCUSSION THREAD

**Discussion by nh43de** [etsy/open-api#722] [Ideas]
### Is there an existing ticket for this?

- [X] I have searched the existing tickets

### Is your feature request related to a problem? Please describe

Related to #96 -

I need the Includes to be able to be specified for findAllListingsActive - is this still in the works?

### Describe the solution you'd like

Ideal solution: ability to specify includes for findAllListingsActive - so that images can be obtained in the same call.

https://developer.etsy.com/documentation/reference/#operation/findAllListingsActive

**Comment by switmac** [comment]
hi @etsyachristensen just want to ask if this is being considered?

**Comment by etsyachristensen** [comment]
It is on the list for review.

**Comment by switmac** [comment]
thank you for the update @etsyachristensen 

will the includes for "Images" "Shop" be included?
I hope so. :)

**Comment by etsyachristensen** [comment]
I would assume that the same "includes" options would be added yes. No guarantees, but that would be my assumption.

**Comment by VasilVelikov** [comment]
One more vote for this issue. I am especially interested in the "Images" association.
Thanks

**Comment by etsyachristensen** [comment]
We are shutting down the issues board, so we're moving this to an Idea Discussion to continue following this.

**Comment by emergingdzns** [comment]
+1 for us on this feature! Please add these to the "find" endpoints.

**Comment by VasilVelikov** [comment]
Hi @etsyachristensen ,
Can you tell us if Etsy plans to develop this feature any time soon? If this feature is not going to be supported anytime in the near future please let us know so we can plan some development efforts to mitigate the problem. I am specifically interested in being able to get the main images of a list of listings.
Thank you!

**Comment by VasilVelikov** [comment]
Hi @nateswart,
Can you please share if this functionality is planned in your roadmap?
There are only 2 months until the deprecation of v2 of the API. The developers need to know if this functionality will be supported in the new version. If it won't be, we need to know some time in advance so we can adjust our functionality in advance.
Thank you!

**Comment by Esatyilmaz0** [comment]
What is the Latest Status?
